### DISCUSSION TITLE
Slightly misleading text about leap seconds

### DISCUSSION THREAD

**Discussion by mhvk** [astropy/astropy-v5.0-paper#111] [Text comments] [answered]
p.13, Time: "... leap seconds had to be manually applied" - this is misleading, leap seconds were always used, but "leap second tables had to be manually updated".  Similarly, in the following sentence, "Now, leap second table updates are done automatically"

**Comment by pllim** [comment, marked as answer]
@mhvk , are you able to PR the suggested changes? Thanks!

**Reply by adrn** [reply]
I updated this in a commit that I will push shortly
